packer_images/ccd_logstash.conf.in

```diff
@@ -8,18 +8,19 @@ input {
         jdbc_driver_library => "/usr/share/logstash/postgresql-42.2.2.jar"
         jdbc_driver_class => "org.postgresql.Driver"
 
-        jdbc_fetch_size => "100"
+        #jdbc_fetch_size => "100" - Removed by Ben Neill to troubleshoot slow indexing 28/06
```

Comment:
you still want this here?

Reply:
will leave it in here until we split per jurisdiction